### DISCUSSION TITLE
Could we do without `column`?

### DISCUSSION THREAD

**Discussion by MPV** [asdf-community/asdf-plugin-manager#28] [Q&A] [answered]
To simplify using this inside Docker images, could we do without `column`?

I know it's mentioned in the README as a dependency:
>https://github.com/asdf-community/asdf-plugin-manager/blob/49d4a448e4ac3c3f1681b6392c1b585be3fc963f/README.md?plain=1#L37-L40

...but while it's standard on most desktop systems, it's not included in most small Linux docker images, hence needing to be installed as part of different bundles. 

Also, its packaging differs between `util-linux` or `bsdmainutils` for different distros, as per: 
- https://command-not-found.com/column 
- googling `column command not found`

One might not need all those tools in your images, as per:
>It provides banner (as printerbanner), calendar, col, colcrt, colrm, column,
from (as bsd-from), hexdump (or hd), look, lorder, ncal (or cal), ul, and
write (as bsd-write).
>-- https://developer.puri.sm/licenses/Librem5/Birch/bsdmainutils.html

**Comment by aabouzaid** [comment, marked as answer]
@MPV thanks for the suggestion :bow:
I remember that I achieved the same result using AWK but I cannot research it now :grin: 
A PR is more than welcome.
